### PR TITLE
Update Slashcommands

### LINKED ISSUE #161: Add interaction data object

### PR DESCRIPTION
**Pull request by Mineinjava**
Resolves #161, #162, makes #154 obselete, makes #155 in progress
added 2 objects, 1 field, and a couple enums
added a bunch of things to the TODO

### CODE REVIEW

File: discordSplash/channel.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 from dataclasses import dataclass
```
Comment (Astr0clad) [inline discordSplash/channel.py:1]: For imports, decide whether there should be a whitespace between them, or no whitespace.

**Review by Astr0clad — CHANGES_REQUESTED** [review]
Fix formatting with the imports in channel.py, note that the other files use no separation whitespace (at least towards what I observed). Otherwise, LGTM.

File: discordSplash/channel.py (hunk 1 above)
Comment (Mineinjava) [reply discordSplash/channel.py:1]: fixed

**Review by Astr0clad — APPROVED** [review]
:)

**Comment by Mineinjava** [comment]
merging this!

**Pull request merged**